ckanext/ozwillo_organization_api/plugin.py

```diff
@@ -115,6 +117,9 @@ def create_organization(context, data_dict):
                          value=client_secret).save()
         session.flush()
 
+        # Automatically add data from data gouv
+        after_create(group, data_dict['organization'])
```

Comment:
in the request received from the provisioning, we also have the datacore id of the organization which contains the siret number. maybe it is a better way to match with the target organization ?

Reply:
The data gouv API does not handle siret numbers so I have to add one more request to get the organization name from the siret (https://www.verif-siret.com/api/documentation/).
Also do all cities have siret ? I couldn't find usefull info about this.
For now this module only handle territories in data gouv like https://www.data.gouv.fr/fr/territories/commune/06083@1942-01-01/Menton/, not organization. Should I add organization management too ?

Reply:
* All organizations created in Ozwillo have a SIRET number (although it may be a fake one, there is no validity checking on Ozwillo side)
* I thought about something like that :
  * From the SIRET number, get the legal name of the organization
  * Try to find a matching territory on data.gouv.fr
  * If one exists, "import" the dataset and resources

The main advantage I see in this process is that we have no side effect with the way the organizations are named in Ozwillo.

Reply:
Ok we can do this, but how can I get the legal name from a SIRET ? With a public api such as https://www.verif-siret.com/api/documentation/ ? What if the SIRET is not found ?

Reply:
* For the second question : we do nothing (probably, the SIRET is not a valid one)
* For the first one, I don't know the options. Did you find other APIs that allow to do that (maybe the SIRENE one ?)

Reply:
We can start with this one. Later, we may consider deploying https://github.com/betagouv/sirene_as_api.

Reply:
Ok so I implemented this with verif-siret but I ran into a problem : when I use the SIRET to get the organization name, I get answers like 'commune de peymeinade' or 'ville de nice', so when i use this name to search for territory in data gouv I get no results because data gouv only knows 'peymeinade' or 'nice'. As we don't always know what kind of answer we will get, it is hard to automatically get the territory number from the verif-siret response. Also I need to know what object (and its structure) is passed from ozwillo to ckan to be able to get the SIRET number.